### ISSUE TITLE
abnormal training loss with 0 success rate in evaluation

### ISSUE THREAD

**Issue by HaoranChen0212** [PKU-HMI-Lab/AC-DiT#6]
<img width="2142" height="633" alt="Image" src="https://github.com/user-attachments/assets/744cf342-5618-4e97-9952-a9d60b2aea76" />
during the two stage training, the loss curve is very unusual, see in above image, After two stage training, when doing evaluation, i got 0 success rate.Success rate: 0.0% What could be the cause, i did not make any logic change to the codespace except to adapt my workstation.

**Comment by CSCSX** [comment]
<img width="2126" height="627" alt="Image" src="https://github.com/user-attachments/assets/a6dad06c-a92a-4d88-9940-0f03ddb6687e" />

Thanks for the report. Our experiments were conducted on **8×A800 GPUs**, and the results in the paper are based on that exact setup.

"Adapting to your workstation" likely involves changes to the number of GPUs, batch size, or gradient accumulation steps, which directly affect training dynamics and are **not trivial modifications**. For reference, we attach our training curves above, which look quite different from yours. This is more consistent with a misconfigured training setup rather than a code bug.

We'd suggest double-checking that your effective batch size and distributed training configuration match the original settings provided in the training scripts in this repository.
